liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -410,13 +406,58 @@ private String getClassesDirectory(String outputDir) {
         return null; // directory does not exist.
     }
 
-    public String getEEVersion(MavenProject project) {
+    /**
+     * Returns the EE major version detected for the given MavenProjects
+     * 
+     * @param mavenProjects project modules, null if a single module project
+     * @return the latest version of EE detected across multiple project modules,
+     *         null if an EE version is not found
+     */
+    public String getEEVersion(List<MavenProject> mavenProjects) {
+        String eeVersion = null;
+        Set<String> eeVersionsDetected = new HashSet<String>();
+        for (MavenProject mavenProject : mavenProjects) {
+            String ver = getEEVersion(mavenProject);
+            if (ver != null) {
+                eeVersionsDetected.add(ver);
+            }
+        }
+        if (!eeVersionsDetected.isEmpty()) {
+            eeVersion = eeVersionsDetected.iterator().next();
+            // if multiple EE versions are found across multiple modules, return the latest
+            // version
+            for (String ver : eeVersionsDetected) {
+                log.warn(ver);
```

Comment:
Did you mean to include this statement? If so could you add some text like "Detected JavaEE version "+ver ?
Or should it be log.debug()?

Reply:
Also did not mean to log this, fixed